pytorch_lightning/accelerators/cpu.py

```diff
@@ -31,6 +31,7 @@ def setup_environment(self, root_device: torch.device) -> None:
             MisconfigurationException:
                 If the selected device is not CPU.
         """
+        super().setup_environment(root_device)
         if root_device.type != "cpu":
```

Comment:
Would a check like this be moved to `is_available`?

Reply:
is_available doesn't take an input arguments

accepting a device makes it more like `is_compatible` which is a different intention than a global availability check